mlos_bench/config/storage/sqlite3.jsonc

```diff
@@ -0,0 +1,18 @@
+// Local SQLite3 database.
+{
+    "class": "mlos_bench.storage.SqlStorage",
+
+    // Apply --config-path resolution to these config parameters:
+    "resolve_path": ["database"],
```

Comment:
A new feature in config_persistence clas loader: look up for file specified in `database` parameter in `--config-path`. This works for all JSON configs including `Environment` and `Optimizer`. (The functionality is a bit intertwined with some other Storage-specific fixes to be extracted as a separate PR, though)